bookstore/tests/test_handlers.py

```diff
@@ -1,6 +1,76 @@
 """Tests for handlers"""
 import pytest
+import logging
+
+from unittest.mock import Mock
+
+from bookstore.handlers import build_handlers, BookstoreVersionHandler
+from bookstore.bookstore_config import BookstoreSettings, validate_bookstore
+from bookstore.clone import BookstoreCloneHandler, BookstoreCloneAPIHandler
+from bookstore.publish import BookstorePublishHandler
+
+from notebook.base.handlers import path_regex
+from tornado.web import Application
+from traitlets.config import Config
+
+log = logging.getLogger('test_handlers')
 
 
 def test_handlers():
     pass
+
+
+def test_build_handlers_all():
+    web_app = Application()
+    mock_settings = {"BookstoreSettings": {"s3_bucket": "mock_bucket"}}
+    bookstore_settings = BookstoreSettings(config=Config(mock_settings))
+    validation = validate_bookstore(bookstore_settings)
+    handlers = build_handlers(log, '/', validation)
+    expected = [
```

Comment:
Probably would be better to put the expected result before the test steps. Expected result. Run the test. Assert test results match expected.

Reply:
Super reasonable…and usually the pattern I prefer — I don't know what I was thinking.

Reply:
It's Friday :wink: